capitalize_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'base_decorator'
+
+class CapitalizeDecorator < Base_decorator
+  def correct_name
+    given_name = @nameable.correct_name
+    given_name.upcase
```

Comment:
- [ ] Instead of returning the correct name in upper case, please consider capitalizing the output, which means, having the first letter uppercase. 🤓 🔧